conf/claims.routes

```diff
@@ -91,7 +91,7 @@ GET         /check-if-claim-was-sent                                uk.gov.hmrc.
 GET         /change-commodity-details                                uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.EnterCommoditiesDetailsController.changeCommoditiesDetails()
 POST        /change-commodity-details                                uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.EnterCommoditiesDetailsController.changeCommoditiesDetailsSubmit()
 
-GET         /select-duties                                          uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.SelectDutiesController.selectDuties()
+GET         /select-duties                                          uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.SelectDutiesController.selectDuties(isAmend:Boolean=false)
```

Comment:
silly question, is this for backward compatibility?

Reply:
This is because I had to put the back button back into the code.